protected/humhub/modules/file/components/FileManager.php

```diff
@@ -111,4 +111,23 @@ public function findStreamFiles($showInStream = true)
         }
     }
 
+    /**
+     * Returns an array of files assigned to the record
+     *
+     * @return array
+     */
+    public function findAssignedToRecord()
```

Comment:
This method should no longer be used or? 

Otherwise this one should return a similar result:
```
$fileList = $comment->fileManager->find()->select('guid')->column();
```

Reply:
> This method should no longer be used or?
That is right I have deleted this method #4396 delete previous commit and change fileList creation as findAll().
in my 6 Commit



> $fileList = $comment->fileManager->find()->select('guid')->column();

I made it like this:

$fileList = $comment->fileManager->findAll();

That is enough in order to understand whether files exist or no during validation.

Reply:
This can be removed since its not in use anymore?